### PR TITLE
[17.0][UPD] copier version 1.20

### PR DESCRIPTION
**Pull request by MohamedOsman7**

### CODE REVIEW

**Comment by MohamedOsman7** [comment]
@pedrobaeza please review

**Comment by pedrobaeza** [comment]
Please check pre-commit CI

**Comment by MohamedOsman7** [comment]
@pedrobaeza

File: mail_activity_board/__manifest__.py [hunk 1]
```diff
@@ -8,7 +8,8 @@
     "development_status": "Beta",
     "category": "Social Network",
     "website": "https://github.com/OCA/social",
-    "author": "SDi, David Juaneda, Sodexis, ACSONE SA/NV, Odoo Community Association (OCA)",
+    "author": """SDi, David Juaneda, Sodexis, ACSONE SA/NV,
+        Odoo Community Association (OCA)""",
```
Comment (pedrobaeza) [inline mail_activity_board/__manifest__.py]: Don't make literal multi-line strings, or the spaces and newlines will be part of the author field:

```suggestion
    "author": "SDi, David Juaneda, Sodexis, ACSONE SA/NV,"
    "Odoo Community Association (OCA)",
```

File: mail_outbound_static/tests/test_ir_mail_server.py [hunk 2]
```diff
@@ -159,7 +159,8 @@ def test_01_from_outgoing_server_domainone(self):
         self.assertEqual(
             used_mail_server,
             expected_mail_server,
-            f"It using {used_mail_server.name} but we expect to use {expected_mail_server.name}",
+            f"""It using {used_mail_server.name}
```
Comment (pedrobaeza) [inline mail_outbound_static/tests/test_ir_mail_server.py]: Incorrect due to the same reason (and all of them).

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by MohamedOsman7** [comment]
@pedrobaeza how can i fix it? Should it then be ignored it with "# noqa: E501"?

**Comment by pedrobaeza** [comment]
I have already told you what is the solution.

**Review by hbrunn — APPROVED** [review]

**Comment by hbrunn** [comment]
@pedrobaeza do you agree this is okay now?

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge patch

**Pull request merged**